central/cve/fetcher/orchestrator.go

```diff
@@ -207,6 +212,17 @@ func (m *orchestratorCVEManager) reconcileCVEs(clusters []*storage.Cluster, cveT
 			version = metadata.GetVersion()
 		case utils.OpenShift:
 			version = metadata.GetOpenshiftVersion()
+		case utils.Istio:
+			versions, err := m.cveMatcher.GetValidIstioVersions(allAccessCtx, cluster)
+			if err != nil || len(versions) < 1 {
+				continue
+			}
+			for v := range versions {
+				if v != "" {
+					versionToClusters[v] = append(versionToClusters[v], cluster)
+				}
+			}
+			continue
 		}
 
 		if version == "" {
```

Comment:
Maybe we should have `versions` as a string slice and loop through it. That way we don't need the `continue` in the `istio` section